js/siteorigin-panels/model/widget.js

```diff
@@ -45,6 +45,8 @@ module.exports = Backbone.Model.extend( {
 			} else {
 				return '';
 			}
+		} else if ( this.has( 'title' ) ) {
```

Comment:
I actually see that it's been called title quite a lot, so I'll let you make this call. I'm actually fine with either title or label. I'd just like to avoid confusion with the title field added by a lot of widgets that's displayed on the frontend.